holopy/inference/interface.py

```diff
@@ -78,7 +78,7 @@ def make_default_model(base_scatterer, fitting_parameters=None):
     if fitting_parameters is None:
         fitting_parameters = base_scatterer.parameters.keys()
     scatterer = parameterize_scatterer(base_scatterer, fitting_parameters)
-    alpha_prior = Uniform(0, 1, guess=0.7, name='alpha')
+    alpha_prior = Uniform(0.5, 1, name='alpha')
```

Comment:
What is the logic for this change here?

Reply:
fits were going to alpha -> 0 if the initial guess wasn't great. I'd prefer a lower bound with more justification if you have one...

Reply:
I'm not sure there is any good reason for anything regarding alpha. But I would say (my 2 cents):
* Keep `guess = 0.7`, since that is typically what we find is a good fit and since that does provide some information to external users as to what to expect for alpha.
* The alpha=0 is a problem, but perhaps we can just set the lower bound a little higher? 0.5 is just a little too close to "typical" values in my opinion, although maybe that is a good thing. 

Just my 2 cents. I'm fine with whatever you decide, now that I know the change was deliberate :)

Reply:
This is in the user-friendly functional interface, so the goal should be to do whatever will just "work" - I think keeping alpha in a typical range is a good thing. These users don't know anything about alpha. I had earlier suggested fixing it alpha at 1.0 for these calculations, but others rightfully pointed out this isn't what tends to work best. I could maybe fix it at 0.7 though or leave it as a uniform prior over [0.5, 1]. This has a guess at 0.75, which I thought was also reasonable, but I can explicitly set it to 0.7 if you think that's better.

Reply:
Ah I forgot that the guess defaults to the midpoint of the interval. This is fine for me!